templates/base.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{% block title %}Taxi Service{% endblock %}</title>
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+
+    <!-- Include the sidebar -->
+    {% include 'includes/sidebar.html' %}
```

Comment:
The sidebar is included correctly using the `{% include 'includes/sidebar.html' %}` tag. This ensures that the sidebar is present on all pages extending this base template.